package.json

```diff
@@ -34,9 +34,11 @@
     "@typescript-eslint/eslint-plugin": "^3.6.1",
     "@typescript-eslint/parser": "^3.6.1",
     "autocannon": "^6.0.0",
-    "graphql-tools": "^6.2.6",
     "docsify-cli": "^4.4.1",
     "fastify": "^3.0.2",
+    "graphql-middleware": "^6.0.2",
+    "graphql-shield": "^7.5.0",
+    "graphql-tools": "^6.2.6",
```

Comment:
I believe the `graphql-tools` change here is my editor doing a _reorder_ or deps there.